### PR TITLE
Fixing empty map, when one of te markers info has a newline character…

### PR DESCRIPTION
**Pull request by PaulBoon**
@DANS-KNAW/easy 

Where to look for the fix:
Module dans-wicket-leaflet-lib method MapViewerPanel.getMarkersJS

How to reproduce the problem:
First look for a dataset (or create one) which shows on te map. 
Edit the EMD with Fedora admin tool, add a newline character to the dataset title field. 
You have to update the Solr index for this dataset with the command line tool. Do not update the dataset using the EASY web interface, that will fix the dataset. 
Check that now the dataset won't show on the map without this fix of the dans-wicket-leaflet-lib module.

### CODE REVIEW

**Comment by PaulBoon** [comment]
Om 'cross site scripting' tegen te gaan kan ik beter hier de html 'escapen'. Dit doe ik nu wel in de EASY code, maar het is beter in deze module te doen, dan kan een client van de module het niet per ongeluk vergeten.
Laat ik dit in ieder geval nog even controleren.

**Comment by PaulBoon** [comment]
Stel dat we de html escaping default aan zetten in de library, dan moeten we vanuit EASY deze eerst weer uitzetten. 
De vraag is of dat echt gaat helpen met voorkomen van script injectie. Gebruikers van de library (programmeurs) moeten na het uitzetten van het escapen zelf bedenken dat ze eventuele input van buiten nog moeten escapen.
Ik denk dat we altijd wel html erin willen, dus dat escapen altijd uitgezet gaat worden, de winst is dan minimaal. 
Ofwel, we laten het zo en doen geen escaping in de library.

**Comment by jo-pol** [comment]
Not part of this change but don't the long lines of the license header cause building problems? 

Anyhow it is awkward to view long lines online on github as the horizontal scroll bar is at the bottom of the page, to see for yourself try https://github.com/DANS-KNAW/dans-wicket-leaflet-lib/blob/f707e83c8278927acb1ceac2cf7b83e1330b0560/lib/src/main/java/nl/knaw/dans/libs/map/MapViewerPanel.java

**Comment by jo-pol** [comment]
I don't know much about the details, but the variable script part looks like data. Wouldn't it be better to separate the code and load the data as json?

**Comment by janvanmansum** [comment]
@jo-pol: The reformatting of the Apache license header is caused by the Java code formatter that is automatically run with the build. We should probably turn it off.

**Comment by janvanmansum** [comment]
@jo-pol, @PaulBoon  : I think loading a json file would not improve the code here, as the coordinates are dynamically injected. I may be wrong, but I don't see an obvious way to improve this.

**Comment by PaulBoon** [comment]
First of all my apologies for using Dutch in the comments. 
Somehow my java code comments are made into long lines when I do a mvn clean install, maybe I do something wrong here?

**Comment by rvanheest** [comment]
What do you mean regarding those comments? Are they like formatted differently in the files itself when you do `mvnci`? Or do you see warnings about the javadoc in the output of your `mvnci`? The latter is something @janvanmansum has been dealing with over the last couple of weeks, I thought. The solution was to remove the links to JDK classes/methods from the javadoc.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* 
+* 
+* 
+
+#### Where should the reviewer @DANS-KNAW/dans-wicket-leaflet-lib start?
+
+#### How should this be manually tested?
+
+* in dans-wicket-leaflet-lib run `mvn clean install`
+* rebuild and run the examples and/or rebuild and deploy easy-webui
```
Comment (jo-pol) [inline .github/PULL_REQUEST_TEMPLATE.md]: Doesn't the first "rebuild" repeat the line above? If not, what do you mean?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+fixes EASY-
+
+#### When applied it will
+* 
+* 
+* 
+
+#### Where should the reviewer @DANS-KNAW/dans-wicket-leaflet-lib start?
```
Comment (rvanheest) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this @dans-knaw/dans-wicket-leaflet-lib a real thing? I don't think so. Should be @DANS-KNAW/easy if you ask me...

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (PaulBoon) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes, mvn clean install of the lib builds the example

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (PaulBoon) [inline .github/PULL_REQUEST_TEMPLATE.md]: Look at your browser address bar: https://github.com/DANS-KNAW/dans-wicket-leaflet-lib

Comment (PaulBoon) [reply .github/PULL_REQUEST_TEMPLATE.md]: OK, use easy for now

**Pull request merged**